torch/_inductor/codegen/cuda/gemm_template.py

```diff
@@ -817,6 +826,9 @@ def filter_op(
         ):
             return None
 
+        if not self._has_tma_epilogue(op):
```

Comment:
Remove (I add a config for this in the next PR)

Reply:
+1 - should we figure out another way here ?

Reply:
Yeah in the next PR, I check if the underlying kernel is a TMA kernel and don't allow fusions if so.